kortex_driver/readme.md

```diff
@@ -213,6 +214,50 @@ Many things have been changed in the ros_kortex repository between versions 1.1.
 * The **/my_robot_name/base_feedback/joint_state** topic is now advertised as **/my_robot_name/joint_state**.
 * The [kortex_driver launch file](launch/kortex_driver.launch) is now located in the `kortex_driver` package instead of the `kortex_bringup` package, which was deleted. Some arguments were added to the file.
 
+
+<a id="multiple"></a>
+## Support for multiple arms
+
+The [kortex_driver launch file](launch/kortex_driver.launch) is primarily used to define one-armed robots. 
+Having more than one arm requires prefixing the joints and links to protect against ambiguity when refering those. 
+To this end, an example has been made to demonstrate running a two-armed robot. 
+The same pattern could be repeated to describe a robot with any number of arms.
+
+The example can be launched with the [kortex_dual_driver launch file](launch/kortex_dual_driver.launch).
+
+The launch file uses the same parameters as the one-armed version, except that most parameters are to be defined individually for each arm (with the **left_** and **right_** prefixes).
+
+The prefixes used in the joints and links names can be changed using the following parameters:
+- `left_prefix`
+- `right_prefix`
+
+The following parameters are to be specified for each robot (using `left_` and `right_` prefixes):
+- `arm`
+- `dof`
+- `vision`
+- `gripper`
+- `ip_address`
+- `username`
+- `password`
+- `cyclic_data_publish_rate`
+- `api_rpc_timeout_ms`
+- `api_session_inactivity_timeout_ms`
+- `api_connection_inactivity_timeout_ms`
+- `default_goal_time_tolerance`
+- `default_goal_tolerance`
+
+
+The folowing parameters are **NOT** to be prefixed:
+- `robot_name`
```

Comment:
as I said, I think this should be prefixed (or test it first to make sure there are no namespace clash!)